src/main/java/codeu/model/data/Conversation.java

```diff
@@ -61,4 +63,16 @@ public String getTitle() {
   public Instant getCreationTime() {
     return creation;
   }
+  /** Time display */ 
+  public String getTime() {
+    LocalDateTime localDate = LocalDateTime.ofInstant(creation, ZoneId.systemDefault());
+    int hr = localDate.getHour();
+    Boolean AM=true;
```

Comment:
Spaces before and after "=". And below you want spaces before and after the ":" in the ternary expression.